evergreen/eg_app/util/validators.py

```diff
@@ -50,3 +52,23 @@ def validate_password(password: str) -> bool:
     :returns: `True` if the password passes requirements, `False` otherwise
     """
     return 12 <= len(password) and len(password) < 256
+
+
+def validate_password_pair(password: str, password_confirmation: str):
+    return hmac.compare_digest(password, password_confirmation) and validate_password(
+        password
+    )
```

Comment:
What is the use of `hmac.compare_digest()` over plain `==`?

If you need it, I'd break the line as
```suggestion
    return hmac.compare_digest(password, password_confirmation)
        and validate_password(password)
```

Also, all these changes have nothing to do with the listed purpose of the PR. They're good changes, but you should rename the PR (or break these out into a new PR entirely if you to be really fancy)

Reply:
Yea now that I look at this again I don't think it is necessary. Just a habit of using hmac when doing authentication.

Reply:
That is just the way black formats it and I don't believe your change is worth the hassle of adding a noqa even though it is prettier that way you suggested.

Reply:
I assume it just tries to break the line as close to the limit as possible, which in this case meant separating the argument. I hope now that both lines are good, it shouldn't try to merge them together and make that mess again, right?

Either way, if hmac isnt needed then its a moot point, with `==` it should all fit

Reply:
I changed from hmac.compare_digest() to ==.